sql/base.sql

```diff
@@ -1028,6 +1036,9 @@ INSERT OR IGNORE INTO RequirementArguments
     (RequirementId                          , Name          , Value)
     VALUES 
     ('REQUIRES_PLOT_HAS_PANTANAL'           , 'FeatureType' , 'FEATURE_PANTANAL'             );
+    
+-- Earth goddess pantheon give +1 Faith for each charming tile (from +1 for each breathtaking)
+UPDATE Modifiers SET SubjectRequirementSetId='PLOT_CHARMING_APPEAL' WHERE ModifierId='EARTH_GODDESS_APPEAL_FAITH_MODIFIER';
 
```

Comment:
you know what's odd is that this is already the case in my version... how did that happen? I dont see it on the master branch, so that's super weird that i have it locally unless i recently added it and forgot. I don't think I added anything without a vote tho.

Reply:
if you can, just revert this commit when you update this PR. I prefer not to have the additional spaces, as the files are already massive and require enough scrolling, and like i said, i apparently already have this change locally.

Reply:
Done